src/main/java/ch/epfl/biop/ij2command/Rois2MeasurementMap.java

```diff
@@ -45,11 +60,20 @@
     public void run() {
         ImagePlus results_imp = R2M( imp );
         results_imp.show();
+        IJ.run(results_imp, "Select All", "");
+        IJ.resetMinAndMax(results_imp);
     }
 
     private ImagePlus R2M(ImagePlus imp ){
         // duplicate the imp for the task
         ImagePlus imp2 = imp.duplicate();
+
+
+        // this is a hack in case the input image is a 32-bit
+        // indeed without adding it, the pixels outside of ROIs were not at 0 but 3.4e38!
+        // TODO find a better way to solve this
+        if (imp.getBitDepth()==32) IJ.run(imp2, "16-bit", "");
```

Comment:
Beurk.... At least put a TODO tag, so that somebody finds it again easily

Reply:
>At least put a TODO tag, so that somebody finds it again easily

What do you mean ? There is a //TODO on the line above that creates the TODO task , cf image below 
![image](https://user-images.githubusercontent.com/8309560/111742889-20f8b780-8889-11eb-9953-be9e4e84aa15.png)

I'm not aware of other king of tags 😭